builtin/providers/azure/provider_test.go

```diff
@@ -11,6 +11,18 @@ import (
 var testAccProviders map[string]terraform.ResourceProvider
 var testAccProvider *schema.Provider
 
+const testAccSecurityGroupName = "terraform-security-group"
+
+// testAccStorageServiceName is used as the name for the Storage Service
+// created in all storage-related tests.
+// It is much more convenient to provide a Storage Service which
+// has been created beforehand as the creation of one takes a lot
+// and would greatly impede the multitude of tests which rely on one.
+// NOTE: the storage container should be located in `West US`.
+var testAccStorageServiceName = os.Getenv("AZURE_STORAGE")
```

Comment:
Not sure if I like the const in this file as normally they (`terraform-security-group` for example) are just part of the test configs. 

Having just a single call to get the `AZURE_STORAGE` from the env and use the var `testAccStorageServiceName` everywhere looks like a nice improvement! And indeed I see no problem with this approach as it's indeed tested to exist before getting to this point...